lib/Account.js

```diff
@@ -11,17 +12,21 @@ function Account(name, balance) {
 /**
  * Withdraws a given amount from this account.
  *
- * @param amount: The amount to be withdrawn from the bank acocunt.
+ * @param amount: The amount to be withdrawn from the bank account.
  */
-Account.prototype.withdraw = function(amount) {
-  // Complete funciton here
+Account.prototype.withdraw = function(amount) 
+{
+  // Complete function here
+  this.balance -= amount;
```

Comment:
Prevent the balance from going negative. So if the balance will be negative, don't do anything.